### DISCUSSION TITLE
Dropdown: Allow function in optionLabel

### DISCUSSION THREAD

**Discussion by lukascernydis** [primefaces/community#2667] [PrimeReact]
Hallo,
I would like to propose a change in Dropdown component, specifically in `optionLabel` signature to allow functions.

Currently, if you try to use function for label formating, it is functional, but typescript compiler will complain about `No overload matches this call.`.

```
<Dropdown
    optionLabel={x => `${x.number} - ${x.name}`}
/>
```

**Comment by melloware** [comment]
Feel free to open a ticket and submit a PR.
